pdns/secpoll-auth.cc

```diff
@@ -69,13 +67,16 @@ int doResolve(const string& qname, uint16_t qtype, vector<DNSResourceRecord>& re
   pw.getHeader()->id=dns_random(0xffff);
   pw.getHeader()->rd=1;
 
-  static vector<ComboAddress> s_servers;
-  vector<ComboAddress> servers = parseResolveConf();
-  if(!servers.empty())
-    s_servers = servers; // in case we chrooted in the meantime
+  if (s_servers.empty()) {
+    L<<Logger::Warning<<"No recursors set, secpoll impossible."<<endl;
+    return RCode::ServFail;
+  }
 
-  if(s_servers.empty())
-    L<<Logger::Warning<<"Unable to poll PowerDNS security status, did not get any servers from resolv.conf"<<endl;
+  string msg ="Doing secpoll, using resolvers: ";
+  for (ComboAddress server : s_servers) {
```

Comment:
nit - you could do const auto& server here, or ComboAddress& - not important here, but important for performance in other places.